cmd/snap/cmd_quota_test.go

```diff
@@ -62,13 +62,19 @@ func makeFakeQuotaPostHandler(c *check.C, body, groupName, parentName string, sn
 		buf, err := ioutil.ReadAll(r.Body)
 		c.Assert(err, check.IsNil)
 
-		var snapNames []string
-		for _, sn := range snaps {
-			snapNames = append(snapNames, fmt.Sprintf("%q", sn))
+		switch action {
+		case "remove":
+			c.Check(string(buf), check.DeepEquals, fmt.Sprintf("{\"action\":\"remove\",\"group-name\":%q}\n", groupName))
+		case "ensure":
+			var snapNames []string
+			for _, sn := range snaps {
+				snapNames = append(snapNames, fmt.Sprintf("%q", sn))
+			}
+			snapsStr := strings.Join(snapNames, ",")
+			c.Check(string(buf), check.DeepEquals, fmt.Sprintf("{\"action\":\"ensure\",\"group-name\":%q,\"parent\":%q,\"snaps\":[%s],\"max-memory\":%d}\n", groupName, parentName, snapsStr, maxMemory))
```

Comment:
```suggestion
			c.Check(string(buf), check.DeepEquals, fmt.Sprintf(`{"action":"ensure","group-name":%q,"parent":%q,"snaps":[%s],"max-memory":%d}`, groupName, parentName, snapsStr, maxMemory))
```

Reply:
Hm may need to expliciltly add `fmt.Sprintf(`....`+"\n",..)`, still I think backticks look better than a bunch of `\"`